src/test/java/duke/task/EventTest.java

```diff
@@ -0,0 +1,18 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    @Test
+    public void TimeParser_WithoutDisk_expectedBehavior() {
```

Comment:
Might want to change your test method names to camel case instead of pascal case?